speech_to_text.py

```diff
@@ -184,6 +184,7 @@ def get_aws(service_name):
         cred = response["Credentials"]
         aws = boto3.resource(
             service_name,
+            region_name=os.environ.get("AWS_REGION"),
```

Comment:
without this change, i got this exception when trying to process the queue (running the docker container locally):

```
% docker run --rm --tty --env-file .env sul-speech-to-text --no-daemon
Traceback (most recent call last):
  File "/app/speech_to_text.py", line 342, in <module>
    main(daemon=args.daemon)
  File "/app/speech_to_text.py", line 24, in main
    job = get_job()
          ^^^^^^^^^
  File "/app/speech_to_text.py", line 51, in get_job
    queue = get_todo_queue()
            ^^^^^^^^^^^^^^^^
  File "/app/speech_to_text.py", line 206, in get_todo_queue
    return get_queue(os.environ.get("SPEECH_TO_TEXT_TODO_SQS_QUEUE"))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/speech_to_text.py", line 216, in get_queue
    sqs = get_aws("sqs")
          ^^^^^^^^^^^^^^
  File "/app/speech_to_text.py", line 185, in get_aws
    aws = boto3.resource(
          ^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/boto3/__init__.py", line 101, in resource
    return _get_default_session().resource(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...snip botocore internal part of stack trace...
  File "/usr/local/lib/python3.11/dist-packages/botocore/regions.py", line 278, in _endpoint_for_partition
    raise NoRegionError()
botocore.exceptions.NoRegionError: You must specify a region.
```

Reply:
Ah, I guess I wasn't seeing that because I have this in my `.env`:

```
AWS_DEFAULT_REGION=us-west-2
```

and `env-example` has:

```
AWS_REGION=us-west-2
```

Maybe we can update `env-example` instead?

Reply:
yup, confirmed that works instead, thanks.  rebuilt the container without this change, reproduced the region error, updated `AWS_REGION` to `AWS_DEFAULT_REGION` in my .env, error went away.

will update `env-example`...